requirements.txt

```diff
@@ -0,0 +1,19 @@
+docutils==0.15.2
+matplotlib==1.5.1
+Jinja2==2.8
+pandas==0.24.2
+six==1.10.0
+sphinx_rtd_theme==0.4.3
+setuptools==20.7.0
+Sphinx==1.7.9
+numpy==1.11.0
+requests==2.9.1
+Pillow==6.1.0
+apex==0.9.10dev
+av==6.2.0
+lmdb==0.97
+mock==3.0.5
+pycocotools==2.0.0
+scipy==1.3.1
+torch==1.2.0
+chartio==6.0.1
```

Comment:
Where is this used?

Reply:
- **torch** is listed as optional dependency in `hubconf.py`

- **scipy** is used in `setup.py`

- **pycocotools** is used in these three files `references/detection/coco_eval.py`, `torchvision/datasets/coco.py`, `references/detection/coco_utils.py`

- **chartio** : I used [pipreqs](https://github.com/bndr/pipreqs) to generate requirements.txt, I can't find where is chartio used.. I checked in sanity_checks.ipynb also.